src/linux/wifi/core/AccessPointControllerLinux.cxx

```diff
@@ -197,7 +285,72 @@ AccessPointControllerLinux::SetProtocol(Microsoft::Net::Wifi::Ieee80211Protocol
     }
 
     // Reload hostapd conf file.
-    return isOk && m_hostapd.Reload();
+    isOk = isOk && m_hostapd.Reload();
+
+    return isOk;
+}
+
+bool
+AccessPointControllerLinux::SetAkmSuites(std::vector<Ieee80211AkmSuite> akmSuites)
+{
+    bool isOk = false;
+
+    // Construct the wpa_key_mgmt hostapd property value.
+    std::string concatenatedWpaKeyMgmtPropertyValue{};
+    for (const auto& akmSuite : akmSuites) {
+        const auto& wpaKeyMgmtPropertyValue = detail::IeeeAkmSuiteToWpaKeyMgmtPropertyValue(akmSuite);
+        concatenatedWpaKeyMgmtPropertyValue += wpaKeyMgmtPropertyValue;
+        concatenatedWpaKeyMgmtPropertyValue += " ";
+    }
```

Comment:
`std::string+=` isn't great for concatenation. For a one-liner it's ok, though even then `std::format()` is probably a bit better. For repeated concatenations like this, `std::ostringstream` will be a lot better.